.github/workflows/onpullrequest.yml

```diff
@@ -0,0 +1,19 @@
+# Actions for pull requests only
+name: zecale-ci-pull-request
+
+on:
+  pull_request:
+
+env:
+  MACOS_BREW_PACKAGES: "pkg-config libomp"
```

Comment:
This one doesn't seem required (maybe a copy/paste residue from Zeth?)

Reply:
removed

Reply:
Thanks